docs/orchestration/kubernetes-yaml/README.md

```diff
@@ -408,6 +409,35 @@ kubectl delete statefulset minio \
 && kubectl delete svc minio-service
 ```
 
+### Deploying on a cluster with no storageclass
```

Comment:
@elsonrodriguez Could you please change this and other related sections/links to `Deploying on cluster nodes with host path` . IMO that will be easier to understand and since we already have [`storage-class`](https://github.com/minio/minio/tree/master/docs/erasure/storage-class) for Minio server, we can avoid confusion by using different wording.

Reply:
Fixed!